station/Classes/Dfu/DfuFlasher.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+import iOSDFULibrary
+
+protocol DfuFlasherOutputProtocol: AnyObject {
+    func ruuviDfuDidUpdateProgress(percentage: Float)
+    func ruuviDfuDidUpdateLog(log: DfuLog)
+    func ruuviDfuDidFinish()
+    func ruuviDfuError(error: Error)
+}
+
+class DfuFlasher: NSObject {
+    private let queue = DispatchQueue(label: "DfuFlasher", qos: .userInteractive)
+    private var dfuServiceInitiator: DFUServiceInitiator!
+    private weak var output: DfuFlasherOutputProtocol?
+    private var firmware: DFUFirmware!
```

Comment:
can we go with optional here? It would be safer, it's not compile-time DI Swinject guaranties.